### DISCUSSION TITLE
Improvement suggestions, round 1

### DISCUSSION THREAD

**Discussion by jallubaba** [albar965/littlenavmap#889] [Ideas]
Thanks for the superb app, I've been using it for years. MSFS is the primary platform currently and these comments may refer to it.

I'd like to see more customization options:

1. Allow default sceneries to be hidden altogether
2. Add a selectable colour identifier to addon sceneries, perhaps an additional outer ring onto map icons
3. Allow less zoom (more area visible) when used with bymaximus/
msfs2020-toolbar-little-nav-map (https://github.com/bymaximus/msfs2020-toolbar-little-nav-map)
4. Add an option to force add stuff similarly to the current Scenery Library Database menu. This is for users running MSFS addon linker, whose addons are (junction) linked into the Community folder. Some the addons could be disable and still show them on LNM.
5. A teleport function would be nice
6. Ability to hide AI traffic and leave online traffic visible. Sea traffic could be selected independently.
7. Add a custom timeout after map locks back to user a/c after panning it
8. Add a "visited airfields" field into the logbook.
9. Allow (at least) the main map window the undocked into a separate window.
10. This goes probably too much outside intention of the project. An ability to create parking spots (for fields not having them) to allow cold starts would be great. I don't have a clue how this would work and there's already a 5 character limitation in the sim.

**Comment by albar965** [comment]
Have a look at the [LNM alpha](https://www.avsim.com/forums/topic/614851-little-navmap-alphadevelopment-279-released-update-two/). There are already a lot of improvements.

> * Allow default sceneries to be hidden altogether

You can already force add-on airport display and then hide soft and hard surface airports in the LNM 2.7 alpha. Same effect.

> * Add a selectable colour identifier to addon sceneries, perhaps an additional outer ring onto map icons

Currently there is one color for all. I can make this color configurable but not for each airport or airport groups.
 
> * Allow less zoom (more area visible) when used with bymaximus/
>   msfs2020-toolbar-little-nav-map (https://github.com/bymaximus/msfs2020-toolbar-little-nav-map)

Sorry, you have to contact the toolbar author about this. The toolbar uses the LNM map interface which allows the same zoom levels as the normal LNM user interface. Maybe another developer could take over this project. The toolbar seems abandoned.

> * Add an option to force add stuff similarly to the current Scenery Library Database menu. This is for users running MSFS addon linker, whose addons are (junction) linked into the Community folder. Some the addons could be disable and still show them on LNM.

Most people enable all add-ons and the reload the scenery library in LNM to have all airports available.
I have to see what can be done.
 
> * A teleport function would be nice

Out of scope for LNM in my opinion. There are other tools to do this or you can use the builtin simulator map to move your aircraft.

> * Ability to hide AI traffic and leave online traffic visible. Sea traffic could be selected independently.

Already done in LNM 2.7 alpha. You can hide online network traffic (dark red, IVAO, VATSIM or the like) and AI separately. The change will come with the next alpha release. Note that LNM cannot see a difference between other online traffic and AI. This might not work for all online tools.

> * Add a custom timeout after map locks back to user a/c after panning it

Already there. See options dialog on page `Simulator Aircraft`.

> * Add a "visited airfields" field into the logbook.

Menu `Logbook` -> `Show Statistics` has this. Or select logbook entries in the logbook search result table. This shows all selected trips with a direct connection. You can also disable the direct connection line in the context menu. This will then show only all visited airports.
It might be slow depending on number of log entries. Better filter by a criteria like aircraft before.

> * Allow (at least) the main map window the undocked into a separate window.

Map window can be undocked once enabled in options on page `Map`. It's a bit fiddly, though.

> * This goes probably too much outside intention of the project. An ability to create parking spots (for fields not having them) to allow cold starts would be great. I don't have a clue how this would work and there's already a 5 character limitation in the sim.

Sorry, not doable. You can use the MSFS SDK tools, X-Plane WED or ADE for P3D to modify airports. 
One thing I could think of is placing the aircraft at an arbitrary start position (on an apron, for example). But not all simulators can place the aircraft at arbitrary positions. X-Plane doesn't do this at all.

Alex

**Reply by yuinma** [reply]
Hi, 

I have a suggestion about flight plan procedure selection. 
Please see the pictures as attached. 
with that we don't need to go to Map window and take aim at that airport small icon to click out that menu. that'll be much convienent.

![image](https://user-images.githubusercontent.com/37968547/189277122-101ecdb5-be88-4cb7-a59e-11b79b46bfeb.png)
![image](https://user-images.githubusercontent.com/37968547/189277136-157c6e4b-6ef5-4a27-aab5-22c8038a9fd7.png)

thank you.
regards**

**Reply by albar965** [reply]
Maybe there is a better solution already implemented (besides using keyboard shortcuts):
Go to options on page `Map` and enable `Show Procedures on Airport Click`. The procedure tab will pop up now if you click on the blue airport link in the flight plan header. It will also pop up if you double click on an airport in the flight plan or on the map.

But adding more items like `Show on map` to the context menu can make sense. I'll keep this in mind.

**Comment by yuinma** [comment]
I got you. It's so helpful.

Thank you very much!

regards~

Alexander Barthel ***@***.***> 于2022年9月9日周五 17:54写道：
